src/app/core/modal/modal.component.ts

```diff
@@ -36,6 +37,7 @@ export class ModalComponent {
         this.setScroll(false);
       }
       this.state = state;
+      ref.detectChanges();
```

Comment:
Tried a couple other things (complete manual change detection, triggering from the error handler instead of here), and I think this is still the cleanest.  Since this component can trigger change-detection on only itself (and children), it shouldn't cost anything to have an extra `detectChanges()` in there, if this modal was triggered by a normal angular zone change.